ci/params/dbVersions/taskcat.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
Are we intending on running this in CI? If so, I believe we need to update the bamboo specs.

Reply:
They are not running in CI, from the PR description:
> (the are now just ran manually - I have tested them and the stacks are working fine)